reference/5.1/Microsoft.PowerShell.Core/About/about_Character_Encoding.md

```diff
@@ -0,0 +1,230 @@
+  **Append** parameter (see below).
+
+- `Export-PSSession` creates UTF-8 files with BOM by default.
+
+- `New-Item -Type File -Value` creates a BOM-less UTF-8 file.
+
+- `Send-MailMessage` uses `Default` encoding by default.
+
+- `Start-Transcript` creates `Utf8` files with a BOM. When the **Append**
+  parameter is used, the encoding can be different (see below).
+
+For commands that append to an existing file:
+
+- `Out-File -Append` and the `>>` redirection operator make no attempt to match
+  the encoding of the existing target file's content. Instead, they use the
+  default encoding unless instructed the **Encoding** is used. You must know
+  the encoding of an existing file's content to append using the same encoding.
+
+- In the absence of an explicit **Encoding** parameter, `Add-Content` detects
+  the existing encoding and automatically applies it to the new content. If the
+  existing content has no BOM, `Default` ANSI encoding is used. The behavior of
+  `Add-Content` is the same in PowerShell Core (v6 and higher) except the
+  default encoding is `Utf8`.
+
+- `Export-Csv -Append` matches the existing encoding when the target file
+  contains a BOM. In the absence of a BOM, it uses `Utf8` encoding.
+
+- `Start-Transcript -Append` matches the existing encoding of files that
+  include a BOM. In the absence of a BOM, it defaults to `Ascii` encoding. This
+  encoding can result in data loss or character corruption when the data in the
+  transcript contains multibyte characters.
+
+For cmdlets that read string data in the absence of a BOM:
+
+- `Get-Content` and `Import-PowerShellDataFile` uses the `Default` ANSI
+  encoding. ANSI is also what the PowerShell engine uses when it reads source
+  code from files.
+
+- `Import-Csv`, `Import-CliXml`, and `Select-String` assume `Utf8` in the
+  absence of a BOM.
+
+## Character encoding in PowerShell Core
+
+In PowerShell Core (v6 and higher), the **Encoding** parameter supports the
+following values:
+
+- `ascii`: Uses the encoding for the ASCII (7-bit) character set.
+- `bigendianunicode`: Encodes in UTF-16 format using the big-endian byte order.
+- `oem`: Uses the default encoding for MS-DOS and console programs.
+- `unicode`: Encodes in UTF-16 format using the little-endian byte order.
+- `utf7`: Encodes in UTF-7 format.
+- `utf8`: Encodes in UTF-8 format.
+- `utf8BOM`: Encodes in UTF-8 format with Byte Order Mark (BOM)
+- `utf8NoBOM`: Encodes in UTF-8 format without Byte Order Mark (BOM)
+- `utf32`: Encodes in UTF-32 format.
+
+PowerShell Core defaults to `utf8NoBOM` for all output.
+
+Beginning with PowerShell 6.2, the **Encoding** parameter also allows numeric
+IDs of registered code pages (like `-Encoding 1251`) or string names of
+registered code pages (like `-Encoding "windows-1251"`). For more information,
+see the .NET documentation for
+[Encoding.CodePage](/dotnet/api/system.text.encoding.codepage?view=netcore-2.2&preserve-view=true).
```

Comment:
Is there a reason to hardlock the link to the .NET Core 2.2 page?

Reply:
Fixed